src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -13,6 +13,7 @@ import {
   BUBBLE_RIGHT_PADDING_ATTACHMENT
 } from './ChatItemAttachmentStyles';
 import ChatItemAttachment from './ChatItemAttachment';
+import {LinkableText} from '../../LinkableText';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `LinkableText` is a good addition to make the text in chat messages clickable. This will improve user experience by allowing users to click on links directly from the chat.
```diff
+ import {LinkableText} from '../../LinkableText';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->